pagopa/io-app
# feat(IT Wallet): [SIW-1471] Add skeumorphic card for the EU Disability Card

## Short description
This PR adds the skeumorphic card for the EU Disability Card 

## List of changes proposed in this pull request
- Added skeumorphic card assets
- Added data rendering
- Fixed incorrect claims order in MDL front card
- Extended `DebugPrettyPrint` component with the ability to share its content via device's sharing capabilities

## How to test
This PR needs to be tested on different screen sizes. 
Navigate to the DC credential details screen, check that the card is displayed correctly.
Alternatively, the component can be tested under **Options > Design System > Cards > Documents**. Scroll down to see the Skumorphic card of the EUDC.

## Preview

https://github.com/user-attachments/assets/2e8fd70d-47dd-4599-9fbe-a05e1540590e
